packages/public/inspection-capture-web/src/PhotoCaptureHUD/PhotoCaptureHUDPreviewSight/PhotoCaptureHUDPreview.styles.ts

```diff
@@ -0,0 +1,62 @@
+import { Styles } from '@monkvision/types';
+
+export const styles: Styles = {
```

Comment:
Change the file structure from this :
```
PhotoCaptureHUDPreviewSight
├── PhotoCaptureHUDPreview.styles.ts
├── PhotoCaptureHUDPreview.tsx
└── components
    ├── PhotoCaptureHUDSightsCounter.tsx
    ├── PhotoCaptureHUDSightsOverlay.tsx
    └── PhotoCaptureHUDSightsSlider.tsx
```
to this :
```
PhotoCaptureHUDPreview
├── PhotoCaptureHUDPreview.styles.ts
├── PhotoCaptureHUDPreview.tsx
├── PhotoCaptureHUDSightCounter
│   ├── index.ts
│   ├── PhotoCaptureHUDSightCounter.styles.ts
│   └── PhotoCaptureHUDSightCounter.tsx
├── PhotoCaptureHUDSightOverlay
│   ├── index.ts
│   ├── PhotoCaptureHUDSightOverlay.styles.ts
│   └── PhotoCaptureHUDSightOverlay.tsx
└── PhotoCaptureHUDSightSlider
    ├── index.ts
    ├── PhotoCaptureHUDSightSlider.styles.ts
    └── PhotoCaptureHUDSightSlider.tsx
```

Each component should have its own style declarations! I would rather have 30 small style files than 1 huuuuuge style file